test/test.js

```diff
@@ -480,6 +480,17 @@ describe('filing-cabinet', function() {
       assert.equal(resolved, `${directory}/node_modules/resolve/index.js`);
     });
 
+    it('resolves a path using the resolve root config', function() {
+      const resolved = cabinet({
+        partial: 'ast',
+        filename: `${directory}/test/ast.js`,
+        directory: `${directory}/fake/sub/dir`,
```

Comment:
Non-blocker: I wonder if it would be better to have a separate webpack config just for this test. It would avoid any potential of breaking other tests that rely on that existing config.